stun-turn/list-post-example/list-post-example.3.x.js

```diff
@@ -0,0 +1,10 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+
+const client = require('twilio')(accountSid, authToken);
+
+client.tokens.create({}, function(err, token) {
```

Comment:
with promises api should look like this: https://github.com/TwilioDevEd/api-snippets/blob/next-gen-node-snippets-rest-part-1/rest/addresses/list-post-example-1/list-post-example-1.3.x.js#L8-L17